### PR TITLE
OSDOCS-3333: add supported AWS instance types

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-3333

**Previews**
ROSA (new content): https://deploy-preview-44154--osdocs.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-service-definition#rosa-sdpolicy-aws-compute-types_rosa-service-definition
Dedicated(updated formatting): https://deploy-preview-44154--osdocs.netlify.app/openshift-dedicated/latest/osd_policy/osd-service-definition.html#aws-compute-types_osd-service-definition

OSD-specific PR: https://github.com/openshift/openshift-docs/pull/44281/

### CODE REVIEW

**Comment by bmcelvee** [comment]
@yuwang-RH would you mind reviewing this PR, please? Thanks!

File: modules/rosa-sdpolicy-account-management.adoc [hunk 1]
```diff
@@ -96,6 +105,28 @@ For additional information, see the link:https://kubernetes.io/docs/tasks/admini
 - C5.24xlarge (96 vCPU, 192 GiB)
 ====
 
+.Storage-optimized compute types
+[%collapsible]
+====
+- i3.large (2 vCPU, 15.25 GiB)
+- i3.xlarge	(4 vCPU, 30.5 GiB)
```
Comment (bmcelvee) [inline modules/rosa-sdpolicy-account-management.adoc:112]: I double checked the spacing here and in the high frequency section. It is correct, but rendering strangely on the diff view.

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Comment by arendej** [comment]
Please remove the .large instances.

**Comment by arendej** [comment]
Please ensure the i3en.* sizes are also listed.

**Review by arendej — CHANGES_REQUESTED** [review]
Please remove the .large instance types from the ROSA files.
Please ensure the i3en.* sizes are also listed.

**Comment by yuwang-RH** [comment]
LGTM~

**Review by arendej — CHANGES_REQUESTED** [review]
c5.xlarge is missing

**Review by arendej — CHANGES_REQUESTED** [review]
r6i.large (2 vCPU, 16 GiB) should be removed.

**Pull request merged**